src/theme/types.ts

```diff
@@ -54,6 +60,8 @@ export interface ThemeSpec {
     [ThemeContextKeys.designTokenScheme]: DesignTokenScheme;
     /** Sets the layout scheme of the theme */
     [ThemeContextKeys.layout]?: ThemeLayout | undefined;
+    /** Sets the resource scheme (e.g. images, text) of the theme */
+    [ThemeContextKeys.resourceScheme]: ResourceScheme | undefined;
```

Comment:
```suggestion
    [ThemeContextKeys.resourceScheme]: ResourceScheme;
```

Reply:
Ah my bad. i made it optional previously. thanks for spotting